.github/workflows/qa.yml

```diff
@@ -89,4 +103,11 @@ jobs:
           exit-code: '1'
           ignore-unfixed: true
           vuln-type: 'os'
-          severity: 'CRITICAL,HIGH'
\ No newline at end of file
+          severity: 'CRITICAL,HIGH'
+      - name: Notify failures on Slack
```

Comment:
same here. in the QA is red on a PR it can not be merged anyway and this part of the pipeline will not be run periodically